docs/src/main/asciidoc/building-native-image.adoc

```diff
@@ -566,6 +566,8 @@ Sample Dockerfile for building with Gradle:
 ----
 ## Stage 1 : build with maven builder image with native capabilities
 FROM quay.io/quarkus/ubi-quarkus-native-image:{graalvm-flavor} AS build
+USER root
+RUN microdnf install findutils
```

Comment:
is find utils necessary for building the native artifact?

Reply:
gradlew ( https://github.com/gradle/gradle/blob/master/gradlew , line 235 ) uses xargs which are not in the image by default.